src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -208,13 +208,11 @@ private void ReplicateToDestination()
                         {
                             while (true)
                             {
-#if DEBUG
```

Comment:
`[Conditional("DEBUG")]` will allow it to be compiled in Release, but not actually be called.